custom_components/toyota_na/base_entity.py

```diff
@@ -21,11 +21,11 @@ def unique_id(self):
     def device_info(self):
         return {
             "identifiers": {(DOMAIN, self.vin)},
-            "name": self.vin,
+            "name": self.vehicle_info["displayModelDescription"],
             "model": f'{self.vehicle_info["modelYear"]} {self.vehicle_info["modelDescription"]}',
             "manufacturer": "Toyota Motor North America",
         }
-    

```

Comment:
What editor do you use? I wonder if it's worth putting together a project .vscode setting file for basic linting and whitespace prefs.

Reply:
Yeah vscode.

Reply:
please help to set up the pref files : )

Reply:
Definitely on my plan 🙂